src/tls_wrappers/openssl/x509_openssl10x.c

```diff
@@ -38,4 +39,59 @@ X509_STORE *X509_STORE_CTX_get0_store(X509_STORE_CTX *ctx)
 	return ctx->ctx;
 }
 
+/* Flags for X509_get_signature_info() */
+/* Signature info is valid */
+#define X509_SIG_INFO_VALID 0x1
+/* Signature is suitable for TLS use */
+#define X509_SIG_INFO_TLS 0x2
+
+static int X509_sig_info_get(const X509 *x, int *mdnid, int *pknid, int *secbits, uint32_t *flags)
+{
+	int siginf_mdnid = NID_undef;
+	int siginf_pknid = NID_undef;
+	int siginf_secbits = -1;
+	uint32_t siginf_flags = 0;
+
+	if (!OBJ_find_sigid_algs(OBJ_obj2nid(x->sig_alg->algorithm), &siginf_mdnid,
+				 &siginf_pknid) ||
+	    siginf_pknid == NID_undef)
```

Comment:
Can we put the line 57 merged with line 56 as "&siginf_pknid) || siginf_pknid == NID_undef)" ?
Current style:
	if (!OBJ_find_sigid_algs(OBJ_obj2nid(x->sig_alg->algorithm), &siginf_mdnid,
				 &siginf_pknid) ||
	    siginf_pknid == NID_undef)
i know your purpose to divide two conditions in left and right of "||". It is my suggestion in format level not is function level.